sdks/python/apache_beam/runners/interactive/interactive_beam.py

```diff
@@ -276,6 +284,21 @@ def show(*pcolls, **configs):
       # PCollection `square` and PCollection `cube`, then visualizes them.
       show(square, cube)
   """
+  flatten_pcolls = []
+  for pcoll_container in pcolls:
+    if isinstance(pcoll_container, dict):
+      flatten_pcolls.extend(pcoll_container.values())
+    elif isinstance(pcoll_container, beam.pvalue.PCollection):
+      flatten_pcolls.append(pcoll_container)
+    else:
+      try:
+        flatten_pcolls.extend(iter(pcoll_container))
+      except TypeError:
+        _LOGGER.warning(
+            'The given pcoll %s is not a dict, an iterable or a PCollection.',
+            pcoll_container)
+        return
```

Comment:
It may be good to throw a ValueError or the TypeError? WDYT?

Reply:
Yes, we could raise a ValueError. Changing it!